Disable Implicit Using instructions is misleading!

I clicked on a link from the default source file so I can disable implicit usings.  As I scrolled down I got really excited when I saw a snippet describing how to disable it.  

```
Disable implicit using directives

If you want to remove this behavior and manually control all namespaces in your project, add [<ImplicitUsings>disable</ImplicitUsings>](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#implicitusings) in the project file.
```


Typical asinine Microsoft documentation tells me to add something without telling me where. I figured it must go into the csproj somewhere.  I added the line and it gave me the error:

```
error MSB4067: The element <ImplicitUsings> beneath element <Project> is unrecognized.
```

Well, I continued to read down the document to find out where the line goes. Lo and behold, I was required to install .NET 5 and convert over to .NET 6!  Thanks, Microsoft for wasting my time!  You could have saved me 10 minutes from excoriating you by not misleading us into thinking it requires only a line change. The process itself is unbelievably dumb, which isn't the fault of this department.  I will excoriate them even more!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 447e23ae-0256-d8c1-4dca-d31fe21602a5
* Version Independent ID: 926db74c-be79-f7cd-a4e9-e156dafa6287
* Content: [C# console app template changes in .NET 6 - .NET](https://docs.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates)
* Content Source: [docs/core/tutorials/top-level-templates.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/top-level-templates.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

--- a/docs/core/tutorials/top-level-templates.md
+++ b/docs/core/tutorials/top-level-templates.md
@@ -71,12 +71,15 @@ If you need `using` directives that aren't implicitly included, you can add them
 
 ### Disable implicit `using` directives
 
-If you want to remove this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#implicitusings) in the project file in the `<PropertyGroup>` element, as shown in the following example:
+If you want to remove this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#implicitusings) to your project file in the `<PropertyGroup>` element, as shown in the following example:
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    ...
+    <ImplicitUsings>disable</ImplicitUsings>
+  </PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>disable</ImplicitUsings>

--- a/docs/core/tutorials/top-level-templates.md
+++ b/docs/core/tutorials/top-level-templates.md
@@ -71,7 +71,19 @@ If you need `using` directives that aren't implicitly included, you can add them
 
 ### Disable implicit `using` directives
 
-If you want to remove this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#implicitusings) in the project file.
+If you want to remove this behavior and manually control all namespaces in your project, add [`<ImplicitUsings>disable</ImplicitUsings>`](../project-sdk/msbuild-props.md#implicitusings) in the project file in the `<PropertyGroup>` element, as shown in the following example:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>disable</ImplicitUsings>
+  </PropertyGroup>
+
+</Project>
+```
 
 ## Global `using` directives
 

